src/modules/AutoinstConfig.rb

```diff
@@ -240,7 +244,7 @@ def find_slp_autoyast
           attrList = SLP.FindAttrs(Ops.get_string(m, "srvurl", ""))
 
           if Ops.greater_than(Builtins.size(attrList), 0)
-            url = Builtins.substring(Ops.get_string(m, "srvurl", ""), 17)
+            url = Builtins.substring(Ops.get_string(m, "srvurl", ""), url_starts_at)
```

Comment:
Maybe it should be a constant. Or you could define "service.autoyast:" as a constant and use size here.